shared/roughtime/roughtime.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The go-ethereum Authors
+// This file is part of the go-ethereum library.
```

Comment:
Can you remove this preamble? This isn't part of go-ethereum, at least from what i could tell.

If it is from go-ethereum, can we use the upstream repo instead of including a copy of the code here?

Reply:
Not from go-ethereum, I removed it